### PR TITLE
Add  PowerShell 7 images

### PR DESCRIPTION
**Pull request by AnatoliB**
Resolves issue #326 

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/3.0/buster/amd64/powershell/powershell7-appservice.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    rm -rf /azure-functions-host/workers/powershell/6 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+# Install extension bundles
+RUN EXTENSION_BUNDLE_VERSION=1.3.3 && \
```
Comment (TsuyoshiUshio) [inline host/3.0/buster/amd64/powershell/powershell7-appservice.Dockerfile:19]: Double check the extension bundle version. Currently they are trying to roll out 2.2.0.

File: host/3.0/buster/amd64/powershell/powershell7-slim.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# Install extension bundles
+RUN EXTENSION_BUNDLE_VERSION=1.3.3 && \
+    EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.3.3_linux-x64.zip && \
+    apt-get update && \
+    apt-get install -y gnupg wget unzip && \
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION/$EXTENSION_BUNDLE_FILENAME && \
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME && \
+    EXTENSION_BUNDLE_VERSION_V2=2.0.1 && \
```
Comment (TsuyoshiUshio) [inline host/3.0/buster/amd64/powershell/powershell7-slim.Dockerfile:27]: Same as above

File: host/3.0/buster/amd64/powershell/powershell7.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# Install extension bundles
+RUN EXTENSION_BUNDLE_VERSION=1.3.3 && \
+    EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.3.3_linux-x64.zip && \
+    apt-get update && \
+    apt-get install -y gnupg wget unzip && \
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION/$EXTENSION_BUNDLE_FILENAME && \
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME && \
+    EXTENSION_BUNDLE_VERSION_V2=2.0.1 && \
```
Comment (TsuyoshiUshio) [inline host/3.0/buster/amd64/powershell/powershell7.Dockerfile:27]: Same

**Review by TsuyoshiUshio — APPROVED** [review]
LGTM. Just double check the Extension Bundle version.

File: host/3.0/buster/amd64/powershell/powershell7-appservice.Dockerfile (hunk 1 above)
Comment (AnatoliB) [reply host/3.0/buster/amd64/powershell/powershell7-appservice.Dockerfile:19]: Thank you. I wanted to intentionally keep it consistent with all the other Dockerfiles in this repo, so we can upgrade bundles in one shot.

**Pull request merged**